backend/collaboration-service/app/src/app.ts

```diff
@@ -4,6 +4,7 @@ import dotenv from "dotenv";
 import cors from "cors";
 import morgan from "morgan";
 import apiRouter from "./routes/api";
+import { HttpStatus } from "./utils/HTTP_Status_Codes";
```

Comment:
this doesn't seem to be used in this file?

Reply:
Resolved in [7bdd59c](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g03/pull/22/commits/7bdd59c052711111a6a6165868ae86be1e17e4bc)